aiida_optimize/_optimization_workchain.py

```diff
@@ -65,6 +65,7 @@ def define(cls, spec):
             cls.create_optimizer,
             while_(cls.not_finished)(cls.launch_evaluations, cls.get_results), cls.finalize
         )
+        spec.output('optimal_process_input', help='Input value of the optimal evaluation process.')
```

Comment:
I'm wondering if we should make this `required=False`, and handle the case where the engine returns `None` (if it can't provide a single input node).
For now I guess it's fine as is, since all engines can provide the input node.